lib/spack/spack/paths.py

```diff
@@ -46,7 +46,12 @@
 mock_packages_path = os.path.join(repos_path, "builtin.mock")
 
 #: User configuration location
-user_config_path = os.path.expanduser('~/.spack')
+# This is now obsolete as it can be customized
+# by setting user_path in a config.yaml file at
+# system scope or a scope with higher precedence;
```

Comment:
I should perhaps reword to "system or site scopes," since those are the only two currently targeted / intentionally supported.

Reply:
I did make that clear in the docs though.

Reply:
fix in latest update